### PR TITLE
API page of Settings menu

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  const localization = useLocalization().settings
+  const repository = useRepository()
+
+  return (
+    <div className={widgetClasses.root}>
+      <Paper elevation={0} className={widgetClasses.container} style={{ padding: 0 }}>
+        <Grid container spacing={1} style={{ padding: '10px' }}>
+          <Grid item xs={12} lg={6} style={{ padding: '1.5rem' }}>
+            <div className={classes.rowContainer}>
+              <span>{localization.apiEndpoint}</span>
```
Comment (taki9) [inline apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:50]: remove span

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+              value={repository.configuration.identityServerUrl}
+              inputProps={{
+                readOnly: true,
+                className: classes.input,
+              }}
+              className={classes.paragraph}
+            />
+          </Grid>
+
+          <Divider orientation="vertical" flexItem />
```
Comment (taki9) [inline apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:78]: should be hidden on lg and below

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+              </Link>
+              <Link target="_blank" href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=dotnet">
+                <img className={classes.logo} src={dotnetlogo} alt="dotnet-logo" width="35" height="35" />
+              </Link>
+              <Link target="_blank" href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=react">
+                <img className={classes.logo} src={reactlogo} alt="react-logo" width="35" height="35" />
+              </Link>
+              <Link target="_blank" href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=redux">
+                <img className={classes.logo} src={reduxlogo} alt="reduc-logo" width="35" height="35" />
+              </Link>
```
Comment (taki9) [inline apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:100]: display: inline-block for the Links

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx (hunk 2 above)
Comment (pusztaienike) [reply apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:78]: It should be md and below (I have tried both), but nice catch :) Thanks
divider

**Review by taki9 — APPROVED** [review]

**Comment by herflis** [comment]
1. Add a title to the icons here (javascript, .net, react, redux)
image
2. Please use a react icon with transparent background here (and it is now a bit compressed or something like that)

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+              <Link
+                className={classes.apiDocsLink}
+                target="_blank"
+                href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=react">
+                <img className={classes.logo} src={reactlogo} alt="react-logo" width="35" height="35" />
+              </Link>
+              <Link
+                className={classes.apiDocsLink}
+                target="_blank"
+                href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=redux">
```
Comment (herflis) [inline apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:113]:
```suggestion
                href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=react-redux">
```

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+            </div>
+            <div style={{ margin: '0 0 16px 0' }}>{localization.learnAboutApi}</div>
+            <Link target="_blank" className={classes.clientLink} href={localization.clientLink}>
+              {localization.clientLink}
+            </Link>
+            <div>
+              <Link
+                className={classes.apiDocsLink}
+                target="_blank"
+                href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=jsclient">
```
Comment (herflis) [inline apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:95]:
```suggestion
                href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=js-snclient">
```

**Review by herflis — CHANGES_REQUESTED** [review]

**Comment by pusztaienike** [comment]
> 1. Add a title to the icons here (javascript, .net, react, redux)
>    image
> 2. Please use a react icon with transparent background here (and it is now a bit compressed or something like that)

DONE
image

**Review by herflis — APPROVED** [review]

File: apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+                  target="_blank"
+                  href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=dotnet">
+                  <img className={classes.logo} src={dotnetlogo} alt="dotnet-logo" width="35" height="35" />
+                </Link>
+              </Tooltip>
+              <Tooltip title="react" placement="bottom">
+                <Link
+                  className={classes.apiDocsLink}
+                  target="_blank"
+                  href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=react">
```
Comment (herflis) [inline apps/sensenet/src/components/settings/api-keys-endpoint-widget.tsx:126]:
```suggestion
                  href="https://docs.sensenet.com/api-docs/basic-concepts?chosenLanguage=reactjs">
```

**Pull request merged**